econml/cate_estimator.py

```diff
@@ -540,7 +543,9 @@ def _get_inference_options(self):
         options.update(auto=LinearModelFinalInference)
         return options
 
-    bias_part_of_coef = False
+    @property
+    def bias_part_of_coef(self):
+        return False
```

Comment:
The duplicate is buggage from past code. I don't want to change in case different places use one or the other. 
This is set here as a default for any class that inherits this mixin. 
If a class wants to change the logic, they can re-impelment the method.
There are some sub-classes that don't overrride this I beleive, but even if not, it's meant for use beyond DML